wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

```diff
@@ -53,20 +31,20 @@ In stable, the accuracy (and judgement) requirements for each [grade](/wiki/Game
 
 Meanwhile, osu!(lazer) has these accuracy cutoffs now:
 
-| Grade | osu! / osu!taiko / osu!mania | osu!catch |
-| :-: | :-- | :-- |
-| SS | 100% | 100% |
-| S | ≥95% | ≥98% |
-| A | ≥90% | ≥94% |
-| B | ≥80% | ≥90% |
-| C | ≥70% | ≥85% |
+| Grade | osu! / osu!taiko | osu!mania | osu!catch |
+| :-: | :-- | :-- | :-- |
+| SS | 100% | 100% | 100% |
+| S | ≥95% (no misses) | ≥95% | ≥98% |
+| A | ≥90% | ≥90% | ≥94% |
+| B | ≥80% | ≥80% | ≥90% |
+| C | ≥70% | ≥70% | ≥85% |
 
 |  |  |
 | :-- | :-: |
 | Breaks backwards compatibility | ![Yes][true] |
-| Classic mod revert support | ![No][false] |
+| Revertable using the Classic mod | ![No][false] |
 | Intentionally changed | ![Yes][true] |
-| Needs further consideration | ![Yes][true] |
+| Needs further consideration | ![No][false] |
 
 ### Hit window edge calculations do not match stable
```

Comment:
Not sure if you want to change this here, but we're pretty much at the point where the added complexity of trying to fix this is beyond anything it's achieving. See https://github.com/ppy/osu/pull/26452 for current progress towards maybe making this happen, but I wouldn't hold out.

Probably fine to not update for now though.